data_storage.rb

```diff
@@ -0,0 +1,82 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+require 'json'
+
+module DataStorage
+  def read_books
+    file = 'books-data.json'
+    data = []
+    if File.exist?(file) && File.read(file) != ''
+      JSON.parse(File.read(file)).each do |element|
+        data.push(Book.new(element['title'], element['author'], id: element['id'].to_i))
+      end
+    end
+    data
+  end
+
+  def save_books(books)
+    data = []
+    books.each do |book|
+      data.push({ id: book.id, title: book.title, author: book.author })
+    end
+    File.write('books-data.json', JSON.generate(data))
+  end
+
+  def read_people
+    data = []
+    if File.exist?('people-data.json') && File.read('people-data.json') != ''
+      JSON.parse(File.read(file)).each do |element|
```

Comment:
- [ ] Seems that we got an error on this line when trying to run the app;

![Screen Shot 2022-03-08 at 17 25 53](https://user-images.githubusercontent.com/67211919/157343029-9effc7c4-e333-41d7-8421-3c215821769b.png)